ciao-controller/instance.go

```diff
@@ -359,8 +359,8 @@ func newConfig(ctl *controller, wl *types.Workload, instanceID string, tenantID
 		userData.Hostname = "cnci-" + tenantID
 	}
 
-	// handle storage resources in workload definition
-	if wl.Storage != nil {
+	// if no instance volumes specified, use those in workload
+	if len(volumes) == 0 && wl.Storage != nil {
```

Comment:
In #974 @mcastelino expected that specifying a volume with the boot_index=0 would make it use that instead of the workload.

I'm happy with additive (or instead of) but it just needs to be made obvious to the user what's going on.

Also it's probably a good idea to move this discussion to #974